src/org/labkey/test/components/domain/DomainFieldRow.java

```diff
@@ -74,9 +74,6 @@ public Input nameInput()
 
     /**
      * selects the field data type.  Note: after the field is initially created, the select will be disabled
-     *
-     * @param columnType
-     * @return
      */
```

Comment:
The signature didn't change, the @param and @return values in the javadocs comment should be filled in.